internal/config/config.go

```diff
@@ -24,6 +24,12 @@ type Metadata struct {
 	Build string   `yaml:"build" json:"build"`
 }
 
+type LaunchBy string
+
+const (
+	LaunchByFailrate LaunchBy = "fail_rate"
```

Comment:
Was there a technical reason for the snake case?

```suggestion
	LaunchByFailrate LaunchBy = "fail rate"
```